lib/rubocop/config_loader_resolver.rb

```diff
@@ -172,6 +172,9 @@ def base_configs(path, inherit_from, file)
     def inherited_file(path, inherit_from, file)
       if remote_file?(inherit_from)
         RemoteConfig.new(inherit_from, File.dirname(path))
+      elsif Pathname.new(inherit_from).absolute?
```

Comment:
I think we should probably add some comments for each branch of the `if`, otherwise it's getting really hard to follow which case maps to what exactly.

Reply:
@bbatsov I've added the comments. Let me know if you see a way to improve them further.

Btw, the 2nd and 4th branch could be merged into `Pathname.new(inherit_from).absolute? || !file.is_a?(RemoteConfig)` but I think the way it's currently written is a bit more clear. And avoids unnecessary `File.expand_path` and `File.dirname` in the case of absolute paths.

Reply:
Agreed.